Source/WebCore/page/FrameConsoleClient.cpp

```diff
@@ -468,11 +466,11 @@ void FrameConsoleClient::screenshot(JSC::JSGlobalObject* lexicalGlobalObject, Re
     }
 
     Vector<JSC::Strong<JSC::Unknown>> adjustedArguments;
+    adjustedArguments.reserveInitialCapacity(arguments->argumentCount());
```

Comment:
This allocates initial capacity that is 1 too small when `target` is `nullptr`, resulting in a vector that is too big when the append overflows the initial capacity. Here is one at to correct that, but maybe you can find a better way.
```suggestion
    adjustedArguments.reserveInitialCapacity(arguments->argumentCount() + !target);
```